src/aragonjs-wrapper.js

```diff
@@ -356,7 +356,9 @@ const initWrapper = async (
 
   const wrapper = new Aragon(daoAddress, {
     provider,
-    defaultGasPriceFn,
+    // Let web3 provider handle gas estimations on mainnet
+    defaultGasPriceFn: () =>
+      getGasPrice({ mainnet: { disableEstimate: true } }),
```

Comment:
Are we going to enable it before merging the PR?

Reply:
Nope, this is meant to be disabled for normal transactions